products/xcp-ng.md

```diff
@@ -21,20 +21,27 @@ auto:
         eol: "Support until"
 
 releases:
+-   releaseCycle: "8.3"
+    releaseDate: 2024-07-12
+    eol: 2028-11-30
+    latest: "8.3.0"
+    latestReleaseDate: 2024-07-12
+    link: https://docs.xcp-ng.org/releases/release-8-3/
```

Comment:
```suggestion
-   releaseCycle: "8.3"
    releaseDate: 2024-10-07
    eol: 2028-11-30
    latest: "8.3.0"
    latestReleaseDate: 2024-10-07
    link: https://docs.xcp-ng.org/releases/release-8-3/
```

Reply:
I was using the release info in this table, and also surprised that it was 07-12

<img width="769" alt="image" src="https://github.com/user-attachments/assets/05338647-7c4d-4214-9a9e-ebf96728b702">